requirements/toolchain.txt

```diff
@@ -1,5 +1,6 @@
 # Tools for installing and setting up the environment.
 
+coverage==4.2
```

Comment:
This is already included in `production.txt` but is now needed outside of of the tox virtualenv.